kolibri/utils/tests/test_cli.py

```diff
@@ -164,7 +164,7 @@ def start_mock(port, *args, **kwargs):
 
         test_port = 1234
 
-        test_zip_port = 5432
+        test_zip_port = 1432
```

Comment:
@rtibbles I'm questioning the lack of previous failures on this test if one had postgres running on 5432. Like prior to these changes I would have expected to see a failure using `make test-with-postgres` but I don't recall that :thinking: Is the assertion on `assert excinfo.code` doing what it needs to?

Reply:
To clarify, this test didn't fail, but updating this seemed to resolve issues with another test.

Reply:
Oh, that is very odd - might be one of the causes of some of the flakiness we see with postgres tests? 

Particlularly odd that it doesn't fail, given that it does explicitly mark the Django DB as needed for the test too.